crates/turborepo-lib/src/turbo_json/mod.rs

```diff
@@ -619,6 +620,11 @@ impl TurboJson {
             TaskName::from("proxy"),
             Spanned::new(RawTaskDefinition {
                 cache: Some(Spanned::new(false)),
+                depends_on: needs_proxy_build.then(|| {
+                    Spanned::new(vec![Spanned::new(UnescapedString::from(format!(
+                        "{MICRO_FRONTENDS_PACKAGE_INTERNAL}#build"
```

Comment:
Don't love that we have to hardcode this task, but we can iterate on it.

Reply:
I also don't love this, but for all other uses of the MFE package that don't have it as a workspace package this will be unnecessary.